sources/install/package_base.sh

```diff
@@ -122,7 +149,8 @@ function install_rvm() {
     # CODE-CHECK-WHITELIST=add-aliases,add-history,add-to-list
     colorecho "Installing rvm"
     gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
-    curl -sSL https://get.rvm.io | bash -s stable --ruby
+    { command -v gpgconf > /dev/null && gpgconf --kill all || :; }
```

Comment:
What does this command do ?

Reply:
kills all gpg processes, I have a gpg process which was waiting and which failed to build the image